### DISCUSSION TITLE
Make HLS closeAfterInactivity configurable?

### DISCUSSION THREAD

**Discussion by c-markwell** [bluenviron/mediamtx#1496] [Questions] [answered]
Would it be possible to make closeAfterInactivity (in internal/core/hls_muxer.go) a config-file option please?

My use-case is streaming my TV tuner card on-demand.  I currently wait a minute after stopping streaming for the tuner to be freed up to then be able to start watching a different channel.

Thanks very much!

**Comment by aler9** [comment, marked as answer]
i've created a feature request here: #1656
